eq-author/src/App/page/Design/Validation/AnswerValidation.js

```diff
@@ -153,20 +138,22 @@ export const SidebarValidation = styled(SidebarButton)`
   }
 `;
 
-class AnswerValidation extends React.PureComponent {
-  state = {
-    startingTabId: null,
-    modalIsOpen: false,
-  };
-
-  constructor(props) {
-    super(props);
-    this.modalId = `modal-validation-${props.answer.id}`;
-  }
+const ValidationButton = ({id, title, children, hasError, onClick}) =>
+    <SidebarValidation
+      key={id}
+      data-test={`sidebar-button-${kebabCase(title)}`}
+      onClick={onClick}
+      hasError={hasError}
+    >
+      {children}
+    </SidebarValidation>;
 
-  handleModalClose = () => this.setState({ modalIsOpen: false });
+function AnswerValidation({answer}) {
```

Comment:
```suggestion
const AnswerValidation = ({answer}) => {
```

Nice one for kicking the class component to the bucket! 😂 Thanks for that. We're slowly trying to phase them out, where we can. So, what you've done here is really useful. 

The only comment I'll make is that we usually try to stick with the ES6 function syntax when writing components (and functions in general, really). I did a similar thing when I first started, too. This is partly because it's the style set from devs in the past so we try to keep things consistent but also as it's a more modern pattern to use and matches help guides and documentation for if we ever need to refer to them.